automation/check-patch.e2e.sh

```diff
@@ -2,7 +2,7 @@
 
 # This script should be able to execute functional tests against Kubernetes
 # cluster on any environment with basic dependencies listed in
-# check-patch.packages installed and docker running.
+# check-patch.packages installed and docker / podman running.
```

Comment:
nit: Since podman has preference can we change this to:
```suggestion
# check-patch.packages installed and podman / docker running.
```

Reply:
ye btw, we can even drop this change if needed, just have the word docker as a general term
wdyt ?

Reply:
Docker is not a general term, we keep like the comment or replace by something like "OCI bin" (but no one understand that)